warehouse/packaging/models.py

```diff
@@ -107,6 +112,13 @@ def __getitem__(self, version):
             raise KeyError from None
 
     @property
+    def latest_release(self):
+        try:
+            return self.releases.order_by('created')[0]
```

Comment:
So if you did this in a loop with [0] it would do a different query for each one? That's interesting. I just assumed it would do a single query and then index in Python, that's fine then.